src/utils.ts

```diff
@@ -28,6 +28,20 @@ export interface IUserConfig {
     port: number;
     useHttps: boolean;
     userDataDir: string | boolean;
+    webRoot: string;

```

Comment:
This is marked as optional in the readme. Should it be optional here? Or is this a case where if the user doesn't provide the webRoot you will fall back to a default and this will always be defined?

Reply:
Yes, this is not set as optional here as the interface specifies all options. In cases where we expect this to be optional the interface is used with Partial<IUserConfig>, and in the other cases it gets a value based on the precedence launch.json > extension setting > default value.